docs/gettingstarted.rst

```diff
@@ -229,7 +229,59 @@ a string or integer. Below is an implementation of this design in the Python cod
   async def test(ctx, optone: str):
     await ctx.send(content=f"Wow, you actually chose {optone}? :(")
 
+Want to restrict access? Setup permissions!
+-------------------------------------------
+
+Slash commands also supports ability to set permissions to allow only certain roles and/or users 
+to run a slash command. Permissions can be applied to both global and guild based commands. They 
+are defined per guild, and each guild can have multiple permissions. Here table that shows the JSON 
+structure of how choices are represented for a single permission:
+
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| **Field**   | **Type**                                   | **Description**                                                                                     |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| id          | int                                        | Snowflake value of type specified. Represents the target to apply permissions on.                   |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| type        | int                                        | An `ApplicationCommandPermissionType`_.                                                             |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| permission  | boolean                                    | true to allow, false, to disallow.                                                                  |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+
+How the type parameter works is very simple. Discord has many ids to represent different things. As you can 
+set permissions to apply for User or Role, `ApplicationCommandPermissionType`_ is used. It's a number and
+following table shows the supported id types for permissions:
+
++-------------------+-----------+
+| **Name**          | **Value** |
++-------------------+-----------+
+| Role              | 1         |
++-------------------+-----------+
+| User              | 2         |
++-------------------+-----------+
+
```

Comment:
JSON response object should additionally be shown here to exemplify the table. Some developers of this library end up using kwarg "plug-and-play" for coding slash commands (as I know I do) so they may use the JSON route as well.